main.py

```diff
@@ -26,7 +26,7 @@
 logger.addHandler(ch)
 
 class CourseLangChain():
-  def __init__(self, pickleFile="vectorstore.pkl", modelFile="model/chinese-alpaca-2-13b.Q8_0.gguf", cli=False) -> None:
```

Comment:
Dont change class preset parameter, if you want to test other model, you should pass through constructor

Reply:
sorry~forgot to change back to the old one(alpaca-2-13b)